install.sh

```diff
@@ -70,7 +70,7 @@ ros_git_get() {
 
```

Comment:
There were some dependancies (git-lfs is what I can think of off the top of my head) should be included in this PR